.github/workflows/ci.yml

```diff
@@ -40,11 +40,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - uses: actions/setup-python@v4
-    # Install the testing dependencies from setup.cfg, including pylint
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
```

Comment:
I think we want to test all support versions

Reply:
not for linting, linting can be done on just 1

Reply:
Oh right